docs/aws/karpenter.cloudformation.yaml

```diff
@@ -0,0 +1,80 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Resources used by https://github.com/awslabs/karpenter
+Parameters:
+  OpenIDConnectIdentityProvider:
+    Type: String
+    Description: "Example oidc.eks.us-west-2.amazonaws.com/id/1234567890"
+Resources:
+  KarpenterControllerRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      RoleName: KarpenterControllerRole
+      Path: /
+      AssumeRolePolicyDocument: !Sub |
+        {
+          "Version": "2012-10-17",
+          "Statement": [{
+            "Effect": "Allow",
+            "Principal": {
+              "Federated": "arn:aws:iam::${AWS::AccountId}:oidc-provider/${OpenIDConnectIdentityProvider}"
+            },
+            "Action": "sts:AssumeRoleWithWebIdentity",
+            "Condition": {
+              "StringEquals": {
+                "${OpenIDConnectIdentityProvider}:aud": "sts.${AWS::URLSuffix}",
+                "${OpenIDConnectIdentityProvider}:sub": "system:serviceaccount:karpenter:karpenter"
+              }
+            }
+          }]
+        }
+  KarpenterControllerPolicy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyName: Karpenter
+      Roles:
+        -
+          Ref: "KarpenterControllerRole"
```

Comment:
Is this intentionally left on a different line than above?

Reply:
Yeah, just how it formatted.